troposphere/static/js/components/modals/instance/InstanceResumeModal.jsx

```diff
@@ -25,38 +28,72 @@ export default React.createClass({
     // ------
     //
 
-    renderBody: function() {
+    renderBody: function(multiple, instances) {
```

Comment:
I need `multiple` to determine the copy-text, so I either pass it in or I declare a new variable with the same logic as done in `render`.